capitalized_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class CapitalizedDecorator < Nameable
```

Comment:
- [ ] In your Readme file, you have broken links. Please make sure everything is working fine to make your Readme professional.